README.md

```diff
@@ -13,3 +14,9 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+Answers:
+
+Asymptotic analysis may be misleading for actual performance because it is detailing very large inputs, so a small input size could cause the determining factor of runtime to be one of the constants that gets dropped when performing an asymptotic analysis. The input could be in the best or worst format for the algorithm, such as recieving an already sorted array for either insertion sort or quick sort, which could vastly change runtime. It also gives performance with respect to input values, not actual runtimes.
```

Comment:
The constants will have an impact regardless of input size. We can analyze best and worst cases separately. Don't know what you mean by "performance with respect to input values".